doc/development.md

```diff
@@ -1,6 +1,6 @@
 # Development
 
-A Go 1.7+ [environment](https://golang.org/doc/install) and the `blkid.h` headers are required.
+A Go 1.11+ [environment](https://golang.org/doc/install) and the `blkid.h` headers are required. Using go 1.10 may work, but isn't directly supported since it does not support go modules.
```

Comment:
Should we directly call out go versions 1.9.7+ & 1.10.3+ specifically as potentially working as they have minimal module  compatability? (https://github.com/golang/go/wiki/Modules#how-are-v2-modules-treated-in-a-build-if-modules-support-is-not-enabled-how-does-minimal-module-compatibility-work-in-197-1103-and-111)

Reply:
I think we actually need 1.10 for other reasons, don't remember what explicitly.

Reply:
https://github.com/coreos/ignition/commit/8fff7535e66c36acbe4e7cc748d21b2f8d4c44bd#diff-354f30a63fb0907d4ad57269548329e3 Is where we bumped it previously. Unfortunately, not much context there.

Reply:
iirc we bumped it to those versions to have the two most recent go releases supported, not for a compat reason.